src/lib/config.js

```diff
@@ -30,7 +30,7 @@ class Config {
     this.timezone = 0;
 
     this.locale = 'enUS';
-    this.os = 'Mac';
+    this.os = 'Win';
```

Comment:
Right. Maybe this would be better as a separate commit, then? It doesn't seem related to fixing multiple realm handling, which is what the rest of your commit deals with.